pkgs/development/r-modules/generic-builder.nix

```diff
@@ -1,10 +1,17 @@
-{ stdenv, R, xvfb_run, utillinux }:
+{ stdenv, R, tzdata, gettext, gfortran, libcxx, xvfb_run, utillinux
+
+, Cocoa, Foundation
+}:
 
 { name, buildInputs ? [], ... } @ attrs:
 
 stdenv.mkDerivation ({
-  buildInputs = buildInputs ++ [R] ++
-                stdenv.lib.optionals attrs.requireX [utillinux xvfb_run];
+  buildInputs = buildInputs ++ [ R tzdata gettext gfortran ] ++
+                stdenv.lib.optionals attrs.requireX [utillinux xvfb_run]
+    ++ stdenv.lib.optionals stdenv.isDarwin [ Cocoa Foundation ];
```

Comment:
Please add that line so that its formatting is consistent with the one above it.